src/sentry/api/serializers/models/user_notifications.py

```diff
@@ -15,7 +15,7 @@ def get_attrs(self, item_list, user, **kwargs):
         notification_option_key = kwargs["notification_option_key"]
         filter_args = {}
 
-        if notification_option_key in ["alerts", "workflow", "email"]:
+        if notification_option_key in ["mail:alert", "workflow:notifications", "mail:email"]:
```

Comment:
I think it didn't work before. I think we didn't notice because the `serialize` function filters out user option rows with no project or organization.
```
data = {}
for uo in attrs:
    if notification_option_key == "reports:disabled-organizations":
	    ...
    elif uo.project is not None:
        data[uo.project.id] = uo.value
    elif uo.organization is not None:
        data[uo.organization.id] = uo.value
return data
```